src/System.Private.ServiceModel/tools/test/Bridge/Bridge/Program.cs

```diff
@@ -16,6 +16,15 @@ namespace Bridge
 {
     internal class Program
     {
+        // Exit codes returned by this executable
+        private const int Bridge_ExitCode_Success = 0;
+        private const int Bridge_ExitCode_Failure = -1;
+        private const int Bridge_ExitCode_AlreadyRunning = 1;
```

Comment:
Maybe we should add a comment here... AlreadyRunning is reserved only for -require command.